### ISSUE TITLE
Implement lightGallery not working? 

### ISSUE THREAD

**Issue by conmen80** [sachinchoolur/lightslider#265]
I try implement lightGallery into lightSlider, but it doesn't expand when click on the image, what is the issue?
 
    `<link rel="stylesheet" href="lightslider.css"/>
     <link rel="stylesheet" href="lightgallery.css"/>
     <script src=jquery.min.js"></script>
     <script src="lightslider.js"></script> 
     <script src="lightgallery.min.js"></script> 
     <script>
     $(document).ready(function() {
          $('#image-gallery').lightSlider({
		gallery:true,
		item:1,
		loop:true,
		thumbItem:9,
		slideMargin:0,
		enableDrag: false,
		currentPagerPosition:'left',
		onSliderLoad: function(el) {
			el.lightGallery({
			   selector: '#imageGallery .lslide'
			});
		}
          });
     });
     </script>

     <div class="item">            
         <div class="clearfix" style="max-width:474px;">
             <ul id="image-gallery" class="gallery list-unstyled">
                 <li data-thumb="img/thumb/cS-1.jpg" class="lslide"> 
                     <img src="img/cS-1.jpg" />
                 </li>
                 <li data-thumb="img/thumb/cS-2.jpg" class="lslide"> 
                     <img src="img/cS-2.jpg" />
                 </li>
                 <li data-thumb="img/thumb/cS-3.jpg" class="lslide"> 
                     <img src="img/cS-3.jpg" />
                 </li>
                 <li data-thumb="img/thumb/cS-4.jpg" class="lslide"> 
                     <img src="img/cS-4.jpg" />
                 </li>
             </ul>
         </div>
     </div>`

**Comment by RonaldPK** [comment]
You have `selector: '#imageGallery .lslide'` , but your list's ID is `image-gallery`.
Also, you don't need to add `class="lslide"`; the lightSlider will do that.
So this is not an issue.

**Comment by yongwc2710** [comment]
Hi guys, I'm still facing the problem. can any one pleas provide a sample code ?
Please find my code as below. Thanks
`<!DOCTYPE html>
<html lang="en">
    <head>
    <title>lightSlider Demo</title>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="description" content="">
    <link rel="stylesheet"  href="../src/css/lightslider.css"/>
    <style>
    	ul{
			list-style: none outside none;
		    padding-left: 0;
            margin: 0;
		}
        .demo .item{
            margin-bottom: 60px;
        }
		.content-slider li{
		    background-color: #ed3020;
		    text-align: center;
		    color: #FFF;
		}
		.content-slider h3 {
		    margin: 0;
		    padding: 70px 0;
		}
		.demo{
			width: 800px;
		}
    </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    <script src="../src/js/lightslider.js"></script> 
    <script>
			 $(document).ready(function() {
			  $('#image-gallery').lightSlider({
			gallery:true,
			item:1,
			loop:true,
			thumbItem:9,
			slideMargin:0,
			enableDrag: false,
			currentPagerPosition:'left',
			onSliderLoad: function(el) {
				el.lightGallery({
				   selector: '#imageGallery'
				});
			}
			  });
		 });
    </script>
</head>
<body>
	<div class="demo">
        <div class="item">            
            <div class="clearfix" style="max-width:474px;">
                <ul id="image-gallery" class="gallery list-unstyled">
                    <li data-thumb="img/thumb/cS-1.jpg" > 
                        <img src="img/cS-1.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-2.jpg"> 
                        <img src="img/cS-2.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-3.jpg"> 
                        <img src="img/cS-3.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-4.jpg"> 
                        <img src="img/cS-4.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-5.jpg"> 
                        <img src="img/cS-5.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-6.jpg"> 
                        <img src="img/cS-6.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-7.jpg"> 
                        <img src="img/cS-7.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-8.jpg"> 
                        <img src="img/cS-8.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-9.jpg"> 
                        <img src="img/cS-9.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-10.jpg"> 
                        <img src="img/cS-10.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-11.jpg"> 
                        <img src="img/cS-11.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-12.jpg"> 
                        <img src="img/cS-12.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-13.jpg"> 
                        <img src="img/cS-13.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-14.jpg"> 
                        <img src="img/cS-14.jpg" />
                         </li>
                    <li data-thumb="img/thumb/cS-15.jpg"> 
                        <img src="img/cS-15.jpg" />
                         </li>
                </ul>
            </div>
        </div>

    </div>	
        <script src="https://cdn.jsdelivr.net/picturefill/2.3.1/picturefill.min.js"></script>
        <script src="../dist/js/lightgallery.js"></script>
        <script src="../dist/js/lightgallery.min.js"></script>
        <script src="../dist/js/lg-fullscreen.js"></script>
        <script src="../dist/js/lg-thumbnail.js"></script>
        <script src="../dist/js/lg-video.js"></script>
        <script src="../dist/js/lg-autoplay.js"></script>
        <script src="../dist/js/lg-zoom.js"></script>
        <script src="../dist/js/lg-hash.js"></script>
        <script src="../dist/js/lg-pager.js"></script>
        <script src="../lib/jquery.mousewheel.min.js"></script>
</body>
`

**Comment by Tiththa** [comment]
@yongwc2710  still found an answer?
